Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.9.2
+WORKDIR /go/src/github.com/rainforestapp/rainforest-cli
+RUN curl -s https://glide.sh/get | sh
+ADD glide.yaml glide.lock ./
```

Comment:
any particular reason why add just those two files, glide install and then copy over the rest?

Reply:
So the layer cache doesn’t get bust every time a file changes. Not really a big deal but it was annoying me 😉.

Reply:
oh good point 👍